### PR TITLE
[DoctrineBridge] Document the new `entityClass` option

### PR DESCRIPTION
**Pull request by ogizanagi**
Waiting for https://github.com/symfony/symfony/pull/15002 to be merged.

### CODE REVIEW

File: reference/constraints/UniqueEntity.rst [hunk 1]
```diff
@@ -164,6 +165,16 @@ The name of the repository method to use for making the query to determine
 the uniqueness. If it's left blank, the ``findBy`` method will be used.
 This method should return a countable result.
 
+entityClass
+~~~~~~~~~~~
+
+**type**: ``string`` **default**: ``null``
+
+The fully-qualified class name (FQCN) of the entity to get the repository used for making the query to determine
```
Comment (chalasr) [inline reference/constraints/UniqueEntity.rst]: Maybe
> The fully-qualified class name (FQCN) of the entity for which to get the repository used to determine the uniqueness

? Or just keep it as is for consistency

Comment (javiereguiluz) [inline reference/constraints/UniqueEntity.rst]: Here is a proposal that says exactly the same but in a different order:

```
By default, the query performed to ensure the uniqueness uses the repository of
the current class instance. However, in some cases, such as when using Doctrine
inheritance mapping, you need to execute the query in a different repository.
Use this option to define the fully-qualified class name (FQCN) of the Doctrine
entity associated with the repository you want to use.
```

**Comment by ogizanagi** [comment]
PR updated according to @javiereguiluz 's suggestion. The PR on the code repository has been merged too.

**Comment by wouterj** [comment]
👍 
status: reviewed

**Comment by javiereguiluz** [comment]
👍

**Comment by wouterj** [comment]
Thanks @ogizanagi! Prefect PR!

I've moved the versionadded directives directly after the title, as this is consistent with the versionadded directives in form types options.

**Pull request merged**